admin/listPurchases/listView.html

```diff
@@ -407,6 +408,34 @@
     `;
 }
 
+function loadSomeOrders(){
+    requestOrdersWithQuery(createQueryString(), (error, orders) => {
+        if(error) {
+            return console.error("Failed to load entries", error);
+        }
+        entries = orders.orders;
+        amountOfEntries = orders.amountOfEntries;
+        options.find(o => o.cls == "option-new-orders").el.click();
+    })
+}
+
+async function requestOrdersWithQuery(queryString, callback){
+    const response = await fetch(`/admin/orders?${queryString}`, {method: "GET"}); 
+    if(response.status == 200){
+        return callback(null, await response.json());
+    }
+    callback({
+        trace: new Error("Error respones from the server on request"),
+        status: response.statusText,
+        response: await response.text()
+    });
+}
+
+function createQueryString(){
```

Comment:
Instead of `createQueryString` just pass an object of query entries.

This abstraction has the same problem as the create query function from the database call. You are doing abstractions that don't contain more information, but less, and add no value in communication. And are at the same abstraction level as what you are abstracting from.

Basically, these abstractions do nothing for performance, readability or structure. Avoid them.

Instead, do something that directly performs what you want. E.g. pass an object of e.g. `{ limit: 2, offset: 100 }` to `requestOrdersWithQuery`, inside which you can do this:

```js
const query = Object.entries(query)
  .map(([ key, value ]) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`)
  .join("&");

const url = "/admin/orders" + query ? `?${query}` : '';
```

or something similar...